knowledge-base/report-server-service-agent-is-not-running.md

```diff
@@ -40,20 +44,47 @@ To make sure that the service is running and that the Report Server application
   
 1\. Open Windows Task manager and look for Telerik.ReportServer.ServiceAgent service - it should be running.
 
-2\. Open Telerik.ReportServer.ServiceAgent.exe.config located at [installation folder]\\Progress\\Telerik Report Server\\Services and look for the address of the service (baseAddress element). Paste this address in the browser to check if the service is functioning.
+2\. Open **Telerik.ReportServer.ServiceAgent.exe.config** file located at [installation folder]\\Progress\\Telerik Report Server\\Services and look for the address of the service (baseAddress element). Paste this address in the browser to check if the service is functioning.
 
-3\. Open server's Web.config located at [installation folder]\\Progress\\Telerik Report Server\\Telerik.ReportServer.Web and check if the address of the service matches the address provided in Telerik.ReportServer.ServiceAgent.exe.config.  
+3\. Open server's **Web.config** file located at [installation folder]\\Progress\\Telerik Report Server\\Telerik.ReportServer.Web and check if the address of the service matches the address provided in Telerik.ReportServer.ServiceAgent.exe.config.  
   
-4\. Enable tracing in the Telerik.ReportServer.ServiceAgent.exe.config, start the service manually, and check the generated log file for errors.
+4\. Enable tracing in the **Telerik.ReportServer.ServiceAgent.exe.config** file, start the service manually, and check the generated log file for errors.
+
+	<configuration>
```

Comment:
How does this code snippet work without starting a snippet in some way?!

Reply:
I was wondering the same thing. The snippet I've added starts and ends with 4 backticks and looks the same. Maybe it's the indentation? Since you too are not sure how it works, I've decided to put backticks on the other two snippets.